largest-series-product/largest_series_product_tests.erl

```diff
@@ -6,3 +6,15 @@ three_test() -> ?assert( 504 =:= largest_series_product:from_string("0123456789"
 five_test() -> ?assert( 15120 =:= largest_series_product:from_string("0123456789", 5) ).
 
 six_test() -> ?assert( 23520 =:= largest_series_product:from_string("73167176531330624919225119674426574742355349194934", 6) ).
+
+all_zeroes_test() -> ?assert( 0 =:= largest_series_product:from_string("0000", 2) ).
+
+all_contain_zeroes_test() -> ?assert( 0 =:= largest_series_product:from_string("99099", 3) ).
+
+too_long_test() -> ?assertException(error, function_clause, largest_series_product:from_string("123", 4) ).
```

Comment:
I'm not 100% sure on whether to test for failure via just letting it crash (as it is written now, the `function_clause`, or to make the code explicitly return `{error, not_enough_digits}` or something. Existing tests use both styles: https://github.com/exercism/xerlang/blob/master/triangle/triangle_tests.erl#L38 vs https://github.com/exercism/xerlang/blob/master/clock/clock_tests.erl#L6

Happy to do it either way.